zwave-classifier.js

```diff
@@ -23,17 +23,27 @@ const COMMAND_CLASS_METER = 50;               // 0x32
 //const COMMAND_CLASS_SWITCH_ALL = 39;        // 0x27
 const COMMAND_CLASS_CONFIGURATION = 112;    // 0x98
 
+const AEOTEC_MANUFACTURER_ID = '0x0086';
+const AEOTEC_ZW096_PRODUCT_ID = '0x0060';
+
 const QUIRKS = [
   {
+    // The Aeotec devices don't seem to notify on current changes, only on
+    // instantaneous power changes. So we exclude this for now. We might be
+    // able to support this by adding a read of current each time we get a
+    // power changed.
     zwInfo: {
-      manufacturerId: '0x0086',
+      manufacturerId: AEOTEC_MANUFACTURER_ID,
     },
     excludeProperties: ['current'],
   },
   {
+    // The Aeotech ZW096 says it supports the MULTILEVEL command class, but
```

Comment:
Aeotec, not Aeotech

Reply:
For some reason my fingers almost always an extra character when I type Aeotec. Usually its a t :)